pkg/controllers/provisioning/scheduling/volumetopology.go

```diff
@@ -32,6 +32,10 @@ import (
 	volumeutil "sigs.k8s.io/karpenter/pkg/utils/volume"
 )
 
+// UnsupportedProvisioners is a map of volume plugins that are not supported. When a pod requests storage using a PersistentVolumeClaim (PVC)
+// that uses a StorageClass with any of these unsupported provisioners, Karpenter will skip scheduling that pod.
+var UnsupportedProvisioners = make(map[string]struct{})
```

Comment:
```suggestion
var UnsupportedProvisioners = sets.New[string]()
```

nit: We should probably use a set here so that we can call `Has()` here